keyboards/handwired/onekey/keymaps/digitizer/keymap.c

```diff
@@ -0,0 +1,26 @@
+#include QMK_KEYBOARD_H
+
+#include "digitizer.h"
+
+#include "math.h"
+
+#ifndef ADC_PIN
```

Comment:
This should go into the config.h, not here, as this won't be picked up by anything but the keymap.c file.

Reply:
I just removed the ADC stuff as I didn't use it anyway.